core-plugins/docs/TPFSAvro-batchsink.md

```diff
@@ -31,6 +31,17 @@ Defaults to formatting partitions such as ``2015-01-01/20-42.142017372000``.
 **timeZone:** The string ID for the time zone to format the date in. Defaults to using UTC.
 This setting is only used if ``filePathFormat`` is not null.
 
+**partitionOffset:** Amount of time to subtract from the pipeline runtime to get the output partition. Defaults to 0m.
+The format is expected to be a number followed by an 's', 'm', 'h', or 'd' specifying the time unit,
+with 's' for seconds, 'm' for minutes, 'h' for hours, and 'd' for days.
+For example, if the pipeline is scheduled to run at midnight of January 1, 2016,
+and the offset is set to '1d', data will be written to the partition for midnight Dec 31, 2015."
+
+**cleanPartitionsOlderThan:** Optional property that configures the sink to delete old partitions older after successful runs.
```

Comment:
what about:
```
Optional property that configures the sink to delete partitions older than a specified date-time after a successful run.
```
Does it happen after each run? If so, I think singular would be better.